_inc/jetpack-deactivate-dialog.js

```diff
@@ -33,19 +33,32 @@
 
 	var body = $( 'body' )[ 0 ];
 
+	var tracksProps = {
+		location: 'plugins',
+		purpose: 'deactivate',
+	};
+
 	deactivateLinkElem.attr( 'href', 'plugins.php#TB_inline?inlineId=jetpack_deactivation_dialog' );
 	deactivateLinkElem.attr( 'title', deactivate_dialog.title );
 	deactivateLinkElem.addClass( 'thickbox' );
 	deactivateLinkElem.html( deactivate_dialog.deactivate_label );
 	deactivateLinkElem.on( 'click', function( e ) {
 		observer.observe( body, { childList: true } );
+		window.jpTracksAJAX.record_ajax_event( 'termination_dialog_open', 'click', tracksProps );
 	} );
 
 	$( '#jetpack_deactivation_dialog_content__button-cancel' ).on( 'click', function( e ) {
```

Comment:
By only watching the cancel button, we're missing out on the other events that close the modal. 
- X button.
- Clicking outside of the box. 

If we can catch all of these, I think it's reasonable to group them all under the same "dialog_close" event.

Reply:
At first I could not capture these events because thickbox is not very extensible. Will try again

Reply:
done! Bonus: also track closing using ESC key